packages/react-jss/src/createHoc.js

```diff
@@ -57,41 +74,49 @@ let managersCounter = 0
 /**
  * Wrap a Component into a JSS Container Component.
  *
+ * IProps: Props of the InnerComponent.
+ * OProps: The Outer props the HOC accepts.
+ *
  * @param {Object|Function} stylesOrCreator
```

Comment:
I think we can remove those jsdoc comments now

Reply:
Will do some cleanup in another PR. Have some small fixes etc. in mind